spitfire/runtime/udn.py

```diff
@@ -13,7 +13,19 @@
 import inspect
 import logging
 
-from spitfire import runtime
+# Avoid extra lookups on critical paths by importing directly.
+from spitfire.runtime import PlaceholderError
+from spitfire.runtime import UDNResolveError
+from spitfire.runtime import UndefinedAttribute
+from spitfire.runtime import UndefinedPlaceholder
+from spitfire.runtime import UnresolvedPlaceholder
+
+# Import the accelerated C module if available.
+try:
+  from spitfire.runtime import _udn
```

Comment:
Will this need a pylint ignore line? I believe this will complain about import not being at the top